ServerCore/Pages/Submissions/SubmitProto.cshtml

```diff
@@ -16,15 +16,22 @@
         width: 100%;
         height: calc(100vh - 265px);
     }
+    #pdf-frame {
+        width: 100%;
+        height: calc(100vh - 285px);
+    }
 </style>
 @if (Model.Puzzle.CustomURL != null)
 {
     <iframe id="puzzle-frame" title="@Model.Puzzle.Name" sandbox="allow-same-origin allow-scripts" src="@PuzzleHelper.GetFormattedUrl(Model.Puzzle, Model.Event.ID)"></iframe>
 }
 else if (Model.PuzzleFile != null)
 {
-    <iframe id="puzzle-frame" title="@Model.Puzzle.Name" sandbox="allow-same-origin allow-scripts" src="@Model.PuzzleFile.Url.AbsoluteUri"></iframe>
+    <object id="pdf-frame" title="@Model.Puzzle.Name" data="@Model.PuzzleFile.Url.AbsoluteUri" type="application/pdf">
```

Comment:
Not important for a prototype, but this should eventually do some extension checking.